sparkmagic/sparkmagic/livyclientlib/livysession.py

```diff
@@ -59,6 +59,12 @@ def __init__(self, http_client, properties, ipython_display,
         super(LivySession, self).__init__()
         assert u"kind" in list(properties.keys())
         kind = properties[u"kind"]
+
+        if not should_heartbeat:
+            properties.pop(u"heartbeatTimeoutInSecond")
```

Comment:
What do you mean by that? Why the "kind" is not in constants file?
In real code, it is guaranteed to be present due to the change in get_session_properties. And I do have a default added for that config value.